tests/test_comms.py

```diff
@@ -76,11 +72,8 @@ def test_init_comms(self):
                 self.assertIsNone(comms._progress_bar_shutdown)
                 self.assertIsNone(comms._progress_bar_complete)
 
-            MockDatetimeNow.RETURN_VALUES = [datetime(1970, 1, 1, 0, 0, 0, 0)]
-            MockDatetimeNow.CURRENT_IDX = 0
-
             with self.subTest('without initial values', ctx=ctx, n_jobs=n_jobs, order_tasks=order_tasks), \
-                    patch('mpire.comms.datetime', new=MockDatetimeNow):
+                    patch('mpire.comms.time.time', return_value=0.0):
```

Comment:
It should suffice to use
```suggestion
                    patch('time.time', return_value=0.0):
```

(This works because you use `import time` and want to patch the `time` function on that module. If instead you were using  `from time import time`, then you _would_ need to patch the new reference to the imported `time` function on the module where you're using it.)

Reply:
Ah yes, you're right. Thnx :)